### PR TITLE
add carrier info to RenewalQueued and StartDateUpdated events

### PR DESCRIPTION
**Pull request by AlexSJ1990**

### CODE REVIEW

File: src/main/java/com/hedvig/notificationService/service/event/EventRequest.kt [hunk 1]
```diff
@@ -56,15 +57,21 @@ data class ContractRenewalQueuedEvent(
     val contractId: String,
     val contractType: String,
     val memberId: String,
-    val renewalQueuedAt: LocalDate
+    val renewalQueuedAt: LocalDate,
+    val carrierWillBeSwitchedOnRenewal: Boolean,
+    val currentCarrier: Carrier,
+    val carrierOnRenewal: Carrier
```
Comment (FredrikAreschoug) [inline src/main/java/com/hedvig/notificationService/service/event/EventRequest.kt:63]: Are we currently sending this?  or should they be nullable for backward compatibility?

**Review by FredrikAreschoug — COMMENTED** [review]
Looks good please have a look at my comment. And if pp us is still using `events/contractRenewalQueued` I think its worthwhile switching to `/event` to be able to remove the deprecated endpoint sometime

File: src/main/java/com/hedvig/notificationService/service/event/EventRequest.kt (hunk 1 above)
Comment (AlexSJ1990) [reply src/main/java/com/hedvig/notificationService/service/event/EventRequest.kt:63]: do we need it to be backwards compatible - as this is just for customer.io?

**Review by meletios — APPROVED** [review]
Sweet 🚀

**Pull request merged**